boards/feather_esp32/definition.json

```diff
@@ -1,12 +1,25 @@
 {
-    "boardName":"adafruit-huzzah-32",
+    "boardName":"feather_esp32",
     "mcuName":"ESP32",
     "mcuRefVoltage": 3.3,
     "VID":"0x239A",
     "PID":"0x8036",
     "displayName":"Adafruit Feather HUZZAH ESP32",
     "productPageURL":"https://www.adafruit.com/product/3405",
     "documentationURL":"https://learn.adafruit.com/adafruit-huzzah32-esp32-feather",
+    "esptool": {
+      "fileSystemSize": "94208",
+      "blockSize": "4096",
```

Comment:
These should not be strings, just decimal numbers. Unless we have a reason to use strings here that i'm not aware of.

Reply:
Resolved in https://github.com/adafruit/Wippersnapper_Boards/pull/39/commits/7eddb8f8999a502bf8887d96d3febcef8f2cfec2